src/main/java/frc/utils/TriFunction.java

```diff
@@ -0,0 +1,7 @@
+package frc.utils;
+
+public interface TriFunction<In1, In2, In3, Out> {
+
+	Out apply(In1 var1, In2 var2, In3 var3);
+
```

Comment:
```suggestion
static class TriValueFuncion<In, Out> extend TriFunction<In, In, In, Out> {}
```
Will save you space every time you need the same 3 vals. Not must.

Reply:
Typo in @Sh-Pe comment